webextensions/native-messaging-host/wix.json

```
@@ -25,8 +25,8 @@
   },
   "choco": {
     "description": "Native Messaging Host for FlexConfirmMail",
-    "project-url": "https://github.com/clear-code/flex-confirm-mail",
+    "project-url": "https://github.com/clear-code/flex-confirm-mail-stable",
```

```suggestion
    "project-url": "https://github.com/FlexConfirmMail/Thunderbird/tree/stable",
```

https://github.com/clear-code/flex-confirm-mail-stable は存在しないURLなので、変更せずにリポジトリそのものを指定するか、上記のようにstableブランチを指定するのが良さそうに思いました。

---

ありがとうございます。ストアのURLと誤認して存在しないURLを設定してしまっていました。